src/dotnet/Chat.UI.Blazor/Components/ChatView/chat-view.css

```diff
@@ -109,10 +109,11 @@ body.hoverable .chat-header-title:hover {
 }
 
 .new-separator {
```

Comment:
Where is 'new-separator' css class used? Separator inside ChatMessage no longer uses it.

Reply:
Yes, i forgot to delete. Thanks.